src/scss/_global.scss

```diff
@@ -50,10 +50,60 @@ $directions: (top, right, bottom, left);
   }
 }
 
+// font size
+@each $name, $size in $space-sizes {
```

Comment:
You should define a new variable like $space-sizes for this scss class generator